website/project/model.py

```diff
@@ -3879,6 +3881,13 @@ def is_rejected(self):
         else:
             return False
 
+    @property
+    def logs(self):
+        return [
+            '{} on {}'.format(v.title(), k)
```

Comment:
This hinges on the answer to https://github.com/CenterForOpenScience/osf.io/pull/5644#discussion_r62885659